Problems/1002/1002.py

```diff
@@ -0,0 +1,37 @@
+#https://www.acmicpc.net/problem/1002
+
+#만나는 점 0개
+#(x2-x1)^2+(y2-y1)^2 의 루트 값  >  r1+r2
+#만나는 점 1개
+#(x2-x1)^2+(y2-y1)^2 의 루트 값  =  r1+r2
+#만나는 점 2개
+#(x2-x1)^2+(y2-y1)^2 의 루트 값  <  r1+r2
+
+T = int(input())
+a =[]
+b =[]
+
+for i in range(T):
+    a = input()
+    b = a.split()
```

Comment:
```suggestion
b = int(input().split())
```
아래 코드 보니까 b만, 그것도 모든 element에 대해 integer casting 하면서 쓰던데
굳이 a를 할당하고, b도 string으로 놔둘 필요가 없음.